test/check-machines-lifecycle

```diff
@@ -430,6 +430,30 @@ class TestMachinesLifecycle(VirtualMachinesCase):
             qemu_img_info = m.execute(f"qemu-img info /var/lib/libvirt/images/{name}-2.img")
             self.assertNotIn("snapshotA", qemu_img_info)
 
+            # Only debian-stable has libvirt lesser than 7.1.0
+            # Add this condition to notify us when the condition below is no longer needed
+            if m.image == "debian-stable":
+                self.assertTrue(m.execute("virsh --version") < "7.1.0")
```

Comment:
assertLessThan() for more useful failures, in general (but it should be quite obvious)

Reply:
Fixed